tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py{36,37,38}-django{22,30}
+    py{36,37,38}-django{22,30,31}
```

Comment:
We also have to update `.travis.yaml` because it runs each tox env in a separate environment.

Reply:
Done in c599d8f